### PR TITLE
Better offset calculations.

### PR DESCRIPTION
**Pull request by drakonen**
These are some fixes made so the timeline works better with Firefox and IE 9 / IE 10

@kaareal @kevinvandijk

### CODE REVIEW

File: docs/examples/time-line.html [hunk 1]
```diff
@@ -101,7 +101,7 @@
             period: period,
             data: data,
             snapGrid: snapGrid,
-            onClick: function () { alert ("click"); }
+            onClick: function () { alert ("click" + new Date(arguments[0])); }
```
Comment (kaareal) [inline docs/examples/time-line.html]: just a note, its better to avoid arguments, unless you strictly need them

Comment (kevinvandijk) [inline docs/examples/time-line.html]: Why not type out all expected arguments in the function declaration? That would be bit clearer for example/documentation purposes imho.
Like:
```javascript
onClick: function(timestamp, and, whatever, else) { alert ("click" + new Date(timestamp)); }
```

File: src/visualization/time-line.js [hunk 2]
```diff
@@ -180,9 +182,8 @@ define(['../core', './hover-mixin', '../lib/utils', './lib/axis-local-time', './
           clearTimeout(timer);
         })
         .on("mousemove", function(e) {
-          var offset = e.offsetX;
-
-          offset -= self.margin.left;
+          var parentOffset = $(this).offset();
+          var offset = e.pageX - parentOffset.left;
```
Comment (kevinvandijk) [inline src/visualization/time-line.js]: Just checking: Isn't this the same as doing `var offset = $(this).position()`? http://api.jquery.com/position/

**Comment by kevinvandijk** [comment]
:+1: :tada:

File: src/visualization/time-line.js (hunk 2 above)
Comment (drakonen) [inline src/visualization/time-line.js]: Does not seem to do what I want it to do.

**Pull request merged**